recipes/scikit-umfpack/meta.yaml

```diff
@@ -0,0 +1,48 @@
+{% set version = "0.2.1" %}
+
+package:
+  name: scikit-umfpack
+  version: {{ version }}
+
+source:
+  fn: scikit-umfpack--{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/s/scikit-umfpack/scikit-umfpack-{{ version }}.tar.gz
+  sha256: 9ebfb4ac4cd1ec545a9211160c65129a77b6e2fab02d8cfa30b10278f5773fd1
+  patches:
+    # osx patch below is needed in release 0.2.1, but was fixed by commit 987af60 in master
+    - setup-osx.patch  # [osx]
```

Comment:
thanks. I didn't know you could get the patch like that.  I had been using `diff -u`.  Pretty sure I initially made the patch here and later found the PR that had already fixed it.